question-servers/freeformPythonLib/prairielearn.py

```diff
@@ -810,6 +810,42 @@ def matlab_to_numpy(a):
             return (None, 'Invalid format (missing square brackets and not a real number).')
 
 
+def latex_array_from_numpy_array(A, presentation_type='f', digits=2):
```

Comment:
It will probably be best to get the `pl-matrix-latex` element merged first, so this function can come from there?